.github/workflows/stale.yml

```diff
@@ -0,0 +1,15 @@
+name: "Close stale PR"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token:  ${{ secrets.FLATHUBBOT_TOKEN }}
+        stale-pr-message: "This PR didn't receive any update in a year and will be automatically closed in 14 days."
```

Comment:
```suggestion
        stale-pr-message: "This PR hasn't receive any update in a year and will be automatically closed in 14 days."
```

Reply:
It should also say something along "Feel free to re-open it if you plan to continue working on that pull request or think it deserves attention from Flathub admins".

Reply:
Done